bin/apn

```diff
@@ -31,6 +31,7 @@ command :push do |c|
   c.option '-P', '--payload PAYLOAD', 'JSON payload for notifications'
   c.option '-e', '--environment ENV', [:production, :development], 'Environment to send push notification (production or development (default))'
   c.option '-c', '--certificate CERTIFICATE', 'Path to certificate (.pem) file'
+  c.option '-r', '--rawcert CERTIFICATE VALUE', 'Value of the certificate (.pem) file'
```

Comment:
Should we write the full parameter name like `raw-certificate`?

Reply:
Yep, will do so!